identity/registry/registration.go

```diff
@@ -18,15 +18,15 @@
 package registry
 
 import (
-	"github.com/mysteriumnetwork/payments/registry"
-	"github.com/mysteriumnetwork/payments/registry/generated"
 	"github.com/ethereum/go-ethereum/accounts/abi/bind"
 	"github.com/ethereum/go-ethereum/accounts/keystore"
 	"github.com/ethereum/go-ethereum/common"
+	"github.com/mysteriumnetwork/payments/registry"
+	"github.com/mysteriumnetwork/payments/registry/generated"
 )
 
-// IdentityRegistry checks whenever given identity is registered
-type IdentityRegistry interface {
+// Registry checks whenever given identity is registered
```

Comment:
Registry of what? It either should be in identity package or old name should be returned. Current import ant usage doesn't sound: import "...../registry" registry.Registry

Reply:
It should belong to Openvpn Adapter in future, we will mote out later

Reply:
This is not package `registry`, but rather parent/child package `identity/registry`. This is why I allias those package then importing:
```
identity_loading "github.com/mysteriumnetwork/node/identity/loading"
identity_registry "github.com/mysteriumnetwork/node/identity/registry"
``` 

So I thought there is not point to repeat name `identity_loading.IdentityRegistry`. But aliasing each time is easy to forget, so I dont know perfect pattern in Golang :/